setup.py

```diff
@@ -164,7 +172,8 @@ def run(self):
         "setuptools",
     ],
     extras_require={
-        "examples": ["aiomonitor", "click", "click-log", "click-completion"]
+        "examples": ["aiomonitor", "click", "click-log", "click-completion"],
+        "typing": ["mypy-protobuf"],
```

Comment:
Wouldn't we actually also require `mypy` here?

Reply:
See 50a2b99